### PR TITLE
Storage admin metrics

### PR DESCRIPTION
**Pull request by reichert621**
Adds internal metrics to track storage usage across apps.

Screenshot 2024-09-03 at 11 33 56 PM

### CODE REVIEW

**Comment by stopachka** [comment]
Exciting!

File: server/src/instant/model/app.clj [hunk 1]
```diff
@@ -49,6 +49,22 @@
...
+  ([params] (get-with-creator-by-ids aurora/conn-pool params))
+  ([conn app-ids]
+   (sql/select conn ["SELECT a.*, u.email AS creator_email
+                      FROM apps a
+                      JOIN instant_users u ON a.creator_id = u.id
+                      WHERE a.id in (select unnest(?::uuid[]))"
+                     (-> app-ids
+                         vec
+                         (with-meta {:pgtype "uuid[]"})
+                         into-array)])))
```
Comment (reichert621) [inline server/src/instant/model/app.clj:62]: not entirely sure why this is all necessary but this is the only way I was able to get this to work :shrug:

File: client/www/pages/intern/storage.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
+import Head from 'next/head';
+import React, { useEffect, useState } from 'react';
+
+import { useIsHydrated } from '@/lib/hooks/useIsHydrated';
+import { useAuthToken } from '@/lib/auth';
+import { jsonFetch } from '@/lib/fetch';
+import config from '@/lib/config';
```
Comment (stopachka) [inline client/www/pages/intern/storage.tsx:7]: nice, our lib functions are getting together!

File: server/src/instant/model/app.clj (hunk 1 above)
Comment (stopachka) [reply server/src/instant/model/app.clj:62]: Haha love the REPL work!

Comment (stopachka) [reply server/src/instant/model/app.clj:62]: I am honestly not sure why it's needed. Once you land can play with it!

**Review by stopachka — APPROVED** [review]
Let's do this!

**Pull request merged**